test/lightning/usage_tracking/worker_test.exs

```diff
@@ -49,31 +50,54 @@ defmodule Lightning.UsageTracking.WorkerTest do
     end
 
     test "persists a report indicating successful submission" do
-      metrics = @metrics
-
       with_mock Client,
         submit_metrics: &mock_submit_metrics_ok/2 do
         Worker.perform(%{})
       end
 
       report = Report |> Repo.one()
 
-      assert %Report{submitted: true, data: ^metrics} = report
+      assert %Report{submitted: true} = report
```

Comment:
similarly to the submit error scenario, it would be great if we checked that the instrumented data was saved.